eng/sendtohelix.proj

```diff
@@ -46,7 +54,8 @@
   <PropertyGroup Condition="'$(HelixCommand)' == '' and '$(TargetGroup)' == 'AllConfigurations'">
     <HelixPreCommands>set DOTNET_CLI_TELEMETRY_OPTOUT=1;set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1;set DOTNET_MULTILEVEL_LOOKUP=0</HelixPreCommands>
 
-    <HelixCommand>%HELIX_CORRELATION_PAYLOAD%\tools\dotnet.exe msbuild %HELIX_CORRELATION_PAYLOAD%\test.msbuild</HelixCommand>
+    <HelixCommand>dotnet msbuild %HELIX_CORRELATION_PAYLOAD%\test.msbuild</HelixCommand>
+    <HelixCommand>$(HelixCommand) /warnaserror</HelixCommand>
```

Comment:
is this /warnaserror a necessary part of this fix? it wasn't changed in the original commit.

Reply:
It wasn't changed as part of the original commit but it was added here: https://github.com/dotnet/corefx/commit/a0e2e123e690b09c09c9d25c3100475b39edf230 -- this is good to have it as it catch since are reported as warnings by NuGet any package downgrade to catch wrong dependencies.